examples/squeezer_bin/squeezer_bin/imp.rs

```diff
@@ -0,0 +1,70 @@
+use gtk::glib;
+use gtk::gsk;
+use gtk::prelude::*;
+use gtk::subclass::prelude::*;
+
+#[derive(Debug)]
+pub struct SqueezerBin {
+    pub keep_aspect_ratio: bool,
```

Comment:
This should be wrapped by a Cell and made into a real property of the widget. See other examples.

This way you could make a checkbox and bind it's active state to the keep-aspect-ratio property.

I think that would make the example more interesting:)

Reply:
Yes, good idea

Reply:
Done looks like this:
![grafik](https://user-images.githubusercontent.com/53372671/141009992-c74af231-a0cc-4ac0-a2b0-139009608097.png)
![grafik](https://user-images.githubusercontent.com/53372671/141010028-d684728d-53e8-4a30-be0c-2f15d48feae6.png)